packages/chakra-ui-core/src/CAspectRatioBox/tests/CAspectRatioBox.test.js

```diff
@@ -15,17 +14,53 @@ const renderComponent = (props) => {
   return render(base)
 }
 
+/**
+ * Not sure if we need jest-emotion
+ *
+ * Get styles from document.styleSheets
+ * @param {String} selector
+ */
+function getElementStyles (selector) {
+  selector = new RegExp(selector)
+  let styles = []
+  let i; let j; let sel = selector
+  for (i = 0; i < document.styleSheets.length; ++i) {
+    for (j = 0; j < document.styleSheets[i].cssRules.length; ++j) {
+      if (sel.test(document.styleSheets[i].cssRules[j].selectorText)) {
+        // let selectorText = document.styleSheets[i].cssRules[j].selectorText
+        let cssText = document.styleSheets[i].cssRules[j].style.cssText
+        styles += cssText
+      }
+    }
+  }
+  return styles
+}
```

Comment:
maybe i should add `getElementStyles` to test-utils
but i'm not sure...

Reply:
Seems like a handy thing to have. I would also use this.

Reply:
moved